set.mm

```diff
@@ -422480,6 +422481,16 @@ Class symbols for functions (e.g., ` abs ` , ` sin ` )
        ` ( V iEdg E ) ` for the edges of a graph ` G = <. V , E >. ` .
        </li>
 
+       <li><b>LaTeX definitions.</b>
+       Each token has a "LaTeX definition" which is used by metamath.exe to
+       output tex files.  When writing LaTeX definitions, contributors should
+       favor simplicity over perfection of the display, and should only use
+       core LaTeX symbols or symbols from the AMS packages (amssymb, amsmath)
```

Comment:
Some latex definitions that I collected in set.mm require an additional package (and indeed produce errors). For example <code>\mathscr{S}_\mathscr{H}</code> needs the <code>mathrsfs</code> package to work.

I was thinking about doing a pull request with the package added, but after reading this I think I should make an issue request.

Reply:
Added to the generated preambles: https://github.com/metamath/metamath-exe/pull/130/commits/f5b8c11c10ee090c178183b36eb2a2888ecd4cc4

Reply:
And addded it in these conventions too.